netket/operator/local_values.py

```diff
@@ -1,45 +1,114 @@
 from .._C_netket.operator import (
-    _local_values_kernel,
     _der_local_values_kernel,
     _der_local_values_notcentered_kernel,
     LocalLiouvillian
 )
 
-from .._C_netket.machine import DensityMatrix 
+from .._C_netket.operator import _local_values_kernel as _c_local_values_kernel
 
 import numpy as _np
+from numba import jit
 
 
-def _local_values_impl(op, machine, v, log_vals, out):
+from .._C_netket.machine import DensityMatrix
 
-    vprimes, mels = op.get_conn(v)
 
-    log_val_primes = [machine.log_val(vprime) for vprime in vprimes]
+def Ising(hilbert, h, J=1.0):
+    """
+    Constructs a new ``Ising`` given a hilbert space, a transverse field,
+    and (if specified) a coupling constant.
+
+    Args:
+        hilbert: Hilbert space the operator acts on.
+        h: The strength of the transverse field.
+        J: The strength of the coupling. Default is 1.0.
+
+    Examples:
+        Constructs an ``Ising`` operator for a 1D system.
+
+        >>> import netket as nk
+        >>> g = nk.graph.Hypercube(length=20, n_dim=1, pbc=True)
+        >>> hi = nk.hilbert.Spin(s=0.5, graph=g)
+        >>> op = nk.operator.Ising(h=1.321, hilbert=hi, J=0.5)
+        >>> print(op.hilbert.size)
+        20
+    """
+    sigma_x = _np.array([[0, 1], [1, 0]])
+    sz_sz = _np.array([[1, 0, 0, 0], [0, -1, 0, 0],
+                       [0, 0, -1, 0], [0, 0, 0, 1]])
+    return GraphOperator(hilbert, siteops=[-h * sigma_x], bondops=[J * sz_sz])
```

Comment:
I think you got a rebase error. 
This is in `/operator/hamiltonian`. You should delete this stuff.